packages/dds/sequence/src/sequence.ts

```diff
@@ -112,6 +112,14 @@ export interface ISharedSegmentSequenceEvents extends ISharedObjectEvents {
 	);
 }
 
+export type LocalReferenceTracker = Pick<
```

Comment:
why can't we just use the segment sequence itself, why do we need Client at all? ideally we never expose client, even pruned like this.